Workbooks/UsageMetrics/Usage - Users/Usage - Users.workbook

```diff
@@ -0,0 +1,710 @@
+{
+  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json",
+  "version": "Notebook/1.0",
+  "isLocked": true,
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "### Define your Cohort\r\nPlace text in the filter below to define a cohort for this blade. You can save your changes to this cohort. The filter you supply will be respected by all charts on the page.\r\n\r\nExample cohort:\r\n\r\n| where client_City == \"Seattle\""
+      },
+      "conditionalVisibility": {
+        "parameterName": "DontShow",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "text - 14"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "3393f364-da6d-45f7-8457-20ab3b738318",
+            "version": "KqlParameterItem/1.0",
+            "name": "Cohort",
+            "type": 1,
+            "value": "",
+            "isHiddenWhenLocked": true
+          }
+        ],
+        "style": "above",
+        "queryType": 0,
+        "resourceType": "microsoft.insights/components"
+      },
+      "name": "parameters - 15"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "ff00e059-7ba9-4dfa-a926-8288cf30fc05",
+            "version": "KqlParameterItem/1.0",
+            "name": "UsersList",
+            "label": "Show",
+            "type": 2,
+            "isRequired": true,
+            "query": "datatable(value:string, text:string)[\r\n'user_Id', 'All Users', \r\n'user_AuthenticatedId', 'Authenticated Users'\r\n]",
+            "value": "user_Id",
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "0afb0089-98e9-479c-b12e-a982870ce6be",
+            "version": "KqlParameterItem/1.0",
+            "name": "Whoused",
+            "label": "Who used",
+            "type": 2,
+            "isRequired": true,
+            "query": "datatable(value:string, text:string)[\r\n'pageViews,customEvents,requests', 'Any Custom Events, Request or Page View', \r\n'pageViews,customEvents', 'Any Custom Events or Page View', \r\n'customEvents','Any Custom Event', \r\n'pageViews','Any Page View', \r\n'requests','Any Request'\r\n]",
+            "value": "pageViews,customEvents",
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "queryType": 0,
+            "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "37b05d6c-ab7c-4190-94d8-4973fcf06c49",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
```

Comment:
why isn't this just using the standard TimeRange parameter type?  none of the values inside this query would get localized, and you wouldn't be able to select it in time range dropdowns? you'd only be able to use it in queries?

Reply:
changed